### PR TITLE
JH Docker Mount Update

### PR DESCRIPTION
**Pull request by ebberg**
This correctly finds Open Data files in JH based on the goofys mounted filesystem in the Docker container.

### CODE REVIEW

File: dcicutils/jh_utils.py [hunk 1]
```diff
@@ -149,11 +149,20 @@ def find_valid_file_or_extra_file(obj_id, format):
...
     # hardcoded so that we can find which volume to look in by file type
     # will need to be updated if more File types are added to proc file bucket
-    if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+    if file_meta['open_data_url']:
+        data_dir = 'open_data'
+        if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+            file_type = 'wfoutput'
+        else:
+            file_type = 'files'
+        open_data_prefix = '{}'.format(file_type)
```
Comment (willronchetti) [inline dcicutils/jh_utils.py:158]: Why not just call this `open_data_prefix` from the start? The string build here is unnecessary.

File: dcicutils/jh_utils.py [hunk 2]
```diff
@@ -149,11 +149,20 @@ def find_valid_file_or_extra_file(obj_id, format):
     home_dir = '/home/jovyan'
     # hardcoded so that we can find which volume to look in by file type
     # will need to be updated if more File types are added to proc file bucket
-    if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+    if file_meta['open_data_url']:
```
Comment (willronchetti) [inline dcicutils/jh_utils.py:152]: This will crash if this field isn't present. You need to use `.get('open_data_url', None)`.

**Review by willronchetti — CHANGES_REQUESTED** [review]

File: dcicutils/jh_utils.py (hunk 1 above)
Comment (ebberg) [reply dcicutils/jh_utils.py:158]: Changing this to be 'fourfront-webprod/{}', that was why it was separate.

File: dcicutils/jh_utils.py (hunk 2 above)
Comment (ebberg) [reply dcicutils/jh_utils.py]: Updated.

**Pull request merged**